commitizen/cli.py

```diff
@@ -62,10 +62,15 @@
                         "action": "store_true",
                         "help": "show output to stdout, no commit, no modified files",
                     },
+                    {
+                        "name": "--write-message-to-file",
```

Comment:
in case we're going with the `pathlib.Path` approach:

```suggestion
                        "name": "--write-message-to-file",
                        "type": Path,
```

Reply:
Go for it